test/src/Command/QueueCountCommandTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace tests\eLife\Bus\Command;
+
+use eLife\Bus\Command\QueueCountCommand;
+use eLife\Bus\Queue\InternalSqsMessage;
+use eLife\Bus\Queue\Mock\WatchableQueueMock;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @covers \eLife\Bus\Command\QueueCountCommand
+ */
+class QueueCountCommandTest extends TestCase
+{
+    /** @var Application */
+    private $application;
+    /** @var QueueCountCommand */
+    private $command;
+    /** @var CommandTester */
+    private $commandTester;
+    /** @var WatchableQueueMock */
+    private $queue;
+
+    /**
+     * @before
+     */
+    public function setup_command()
+    {
+        $this->application = new Application();
+        $this->queue = new WatchableQueueMock();
+        $this->command = new QueueCountCommand(
+            $this->queue
+        );
+        $this->application->add($this->command);
+        $this->commandTester = new CommandTester($command = $this->application->get($this->command->getName()));
+    }
+
+    /**
+     * @test
+     */
+    public function it_will_count_queue_items() {
+        $this->commandTester->execute(['command' => $this->command->getName()]);
+        $display = trim($this->commandTester->getDisplay());
+        $this->assertEquals('0', $display);
+        $message = new InternalSqsMessage('article', '42');
```

Comment:
rather than overwriting the same variable multiple times, I would inline `new InternalSqsMessage(...)` into the `$this->queue->enqueue(...)` call